config_src/coupled_driver/ocean_model_MOM.F90

```diff
@@ -1096,4 +1115,75 @@ subroutine ocean_public_type_chksum(id, timestep, ocn)
 
 end subroutine ocean_public_type_chksum
 
+!> This subroutine gives a handle to the grid from ocean state
+subroutine get_ocean_grid(OS, Gridp)
+  ! Obtain the ocean grid.
+  type(ocean_state_type) :: OS
```

Comment:
This arguments should be documented. I'm not sure why this wasn't flagged by the checks.